### PR TITLE
Adjust logo and navbar for desktop view

### PR DESCRIPTION
**Pull request by hamzaw9**
Morning session code review exercise.

### CODE REVIEW

**Review by RileyManda — COMMENTED** [review]
# Hi @hamzaw9  👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  UI is looking on point ✔
- Met 99% of the requirements ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css
Comment (RileyManda) [inline style.css:1]: There seems to be an issue with you linters.
Please see attached screen shot and fix the linter errors in your CSS.
Screen Shot 2023-05-11 at 9 19 45 AM

File: style.css [hunk 1]
```diff
@@ -55,16 +59,20 @@ body {
...
   font-size: 16px;
   line-height: 24px;
 }
 
+/** main and about section h3 tag have same style **/
+
 .main h3,
-.about-myself h3 {     /** main and about section h3 tag have same style **/
+.about-myself h3 {
   color: #7f8cff;
```
Comment (RileyManda) [inline style.css:76]: Try to use var(--colors-name) to avoid repetition of colors:

**Review by Pi1998 — COMMENTED** [review]
Hi @hamzaw9  , 

There is nothing else to say :shipit: 
Congratulations! 🎉

## Highlights
n/a
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**